### DISCUSSION TITLE
[Feature] Share photos from other apps directly to Immich

### DISCUSSION THREAD

**Discussion by hussainalhaddad** [immich-app/immich#7746] [Feature Request]
### The feature

Hi

It would be great if the mobile app can handle the ACTION_SEND intent on Android sharesheet, that way we could share a single or multiple photos/videos from other apps directly to Immich without the need to keep them in  a synced folder

![Screenshot_20240308_175451_IntentResolver](https://github.com/immich-app/immich/assets/55337126/9973a6d5-9ecb-476b-8eb1-8aba17c7e83b)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by shenlong-tanwen** [comment]
Duplicate of #2659
